### PR TITLE
CP-49381 Update README for known issue

### PR DESCRIPTION
**Pull request by acefei**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,33 +7,35 @@ This repository is the [Terraform Provider of XenServer](https://registry.terraf
...
+### Prepare Development Environment
 
-If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (see [Requirements](#requirements) above).
+The following instructions are based on Linux system.
 
-To build the provider, you'll need to prepare the local XenServer module for Go. 
-- [Download](https://www.xenserver.com/downloads) the XenServer SDK zip package and unzip
-- Create `goSDK/` directory under `terraform-provider-xenserver/`
-- Copy all source files under `XenServer-SDK/XenServerGo/src/` to `terraform-provider-xenserver/goSDK/` folder
+1. Install Go on your machine
```
Comment (xueqingz) [inline README.md:21]: Already said on line 14.

File: README.md [hunk 2]
```diff
@@ -44,10 +46,12 @@ Set up log with `github.com/hashicorp/terraform-plugin-log/tflog`. To enable log
 
 ```shell
 export TF_LOG_PROVIDER="DEBUG"
+export TF_LOG_PATH=/tmp/terraform.log
 ```
-See https://developer.hashicorp.com/terraform/plugin/log/managing.
 
-### Test
+Please find more details in https://developer.hashicorp.com/terraform/plugin/log/managing.
```
Comment (xueqingz) [inline README.md:52]: Find more details in [Managing Log Output](https://developer.hashicorp.com/terraform/plugin/log/managing).

File: docs/resources/snapshot.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ resource "xenserver_snapshot" "snapshot" {
 
 - `revert` (Boolean) Set to `true` if you want to revert this snapshot to VM, default to be `false`.
 
--> **Note:** When `revert` is true, the snapshot resource will be updated with new configuration first and then revert to VM.
+-> **Note:** `revert` only works after the snapshot resource created. When `revert` is true, the snapshot resource attributes will be updated first(eg. name_label) and then revert to VM.
```
Comment (xueqingz) [inline docs/resources/snapshot.md:91]: Sorry, here should be:
.... updated first, for example `name_label`. And then ...

File: examples/pool-main/main.tf [hunk 4]
```diff
@@ -16,6 +16,25 @@ provider "xenserver" {
   password = local.env_vars["XENSERVER_PASSWORD"]
 }
 
+# get the existing supporter host
```
Comment (xueqingz) [inline examples/pool-main/main.tf:19]: hosts

File: examples/pool-main/main.tf
Comment (xueqingz) [inline examples/pool-main/main.tf:1]: This file will not show on registry website document, if you want to change the example for pool resource, use  examples/resources/xenserver_pool/resource.tf

Comment (acefei) [reply examples/pool-main/main.tf:1]: I don't want to change example, this change only for verify if join and eject operation are able to work together.

File: xenserver/snapshot_resource.go [hunk 5]
```diff
@@ -59,7 +59,7 @@ func (r *snapshotResource) Schema(_ context.Context, _ resource.SchemaRequest, r
 			},
 			"revert": schema.BoolAttribute{
 				MarkdownDescription: "Set to `true` if you want to revert this snapshot to VM, default to be `false`." +
-					"\n\n-> **Note:** When `revert` is true, the snapshot resource will be updated with new configuration first and then revert to VM." +
+					"\n\n-> **Note:** `revert` only works after the snapshot resource created. When `revert` is true, the snapshot resource attributes will be updated first, for example name_label. And then revert to VM." +
```
Comment (xueqingz) [inline xenserver/snapshot_resource.go:62]: `name_label`

**Review by xueqingz — APPROVED** [review]

**Review by xihuan-citrix — APPROVED** [review]

**Pull request merged**